src/Oscoin/Data/Ledger.hs

```diff
@@ -24,12 +24,17 @@ import           Oscoin.Prelude
                  , Word8
                  , div
                  , fromIntegral
+                 , liftA2
                  , map
                  , maxBound
                  , mod
+                 , show
                  , sum
                  , ($)
+                 , (++)
                  , (.)
+                 , (<$>)
+                 , (<*>)
```

Comment:
:eyes:

Reply:
Yeah :thinking: the intent was to um keep track of what we used since the "contracts" are implemented in here, but it's a bit annoying because there's other stuff in this module too, and it's difficult to split the module apart.